### PR TITLE
ROC-3484 Fix template to show evidence type listed even if no description

### PR DESCRIPTION
**Pull request by rattur**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-3484

### Change description

Update evidence template to not show any evidence data with no description

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x ] No

### CODE REVIEW

File: src/main/features/response/views/evidence.njk [hunk 1]
```diff
@@ -27,12 +27,12 @@
             </thead>
             <tbody>
             {% for item in evidence.rows %}
-              <tr>
-                <td>{{ item.type.displayValue }}</td>
-                  {% if item.description %}
-                    <td>{{ item.description }}</td>
-                  {% endif %}
-              </tr>
+              {% if item.description %}
```
[the remark this answers is not in the document]
Comment (rattur) [reply src/main/features/response/views/evidence.njk]: Just tested locally with various scenarios and we correctly display the evidence type with or without the optional description. I have attached a screenshot. Hope this helps @geek-soft 

screen shot 2018-04-04 at 10 53 15

[the remark this answers is not in the document]
Comment (rattur) [reply src/main/features/response/views/evidence.njk]: Just approved the scenarios with the business

File: package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "test:coverage-unit": "TS_NODE_FAST=true NODE_ENV=mocha LOG_LEVEL=OFF nyc mocha --opts mocha.opts $(find src/test \\( -name '*.ts' \\) ! -path '*a11y*' ! -path '*/routes/*')",
     "test:codacy-upload": "cat ./coverage-report/lcov.info | codacy-coverage",
     "test:codecov-upload": "codecov -f ./coverage-report/lcov.info",
-    "test:smoke": "./bin/run-smoke-tests.sh",
+    "test:smoke": "echo disabled due to testing breaking change to journey",
```
Comment (geek-soft) [inline package.json]: Why did you disable it?

Comment (rattur) [reply package.json]: That's strange I didn't make any changes to that file

Comment (geek-soft) [reply package.json]: You merged demo into your branch. Watch more carefully what Git operations you doing. Check PR before passing it for review.

**Pull request merged**